cmd/minishift/cmd/openshift/version.go

```diff
@@ -32,8 +32,8 @@ import (
 // version command represent current running openshift version and available one.
 var versionCmd = &cobra.Command{
 	Use:   "version [command] [flags]",
-	Short: "Print the current running openshift version to stdout",
-	Long:  `Print the current running openshift version to stdout`,
+	Short: "Prints the current running OpenShift version to the console.",
```

Comment:
Actually we need to be careful here. Technically stdout is actually better than console, since you might be re-directing your stdout in which case nothing comes out to the console.